webcompat/helpers.py

```diff
@@ -100,6 +100,18 @@ def get_user_info():
         session['avatar_url'] = gh_user.get('avatar_url')
 
 
+def version_string(json_section):
```

Comment:
Maybe just `dict` with arg comment about `major`, `minor`, and `patch`?

I first didn't add `get_` to function name because I thought it can be confused with other `get_` functions, that "get" as "extract" some info from protocol objects. While our function is just an ancillary one.